### ISSUE TITLE
Copyright clarification: GPL-2 or GPL-2+

### ISSUE THREAD

**Issue by charles2910** [neomutt/neomutt#4181]
Hey there,

I'm packaging the new version of neomutt for Debian and saw the changes in the copyright notice. Since I'm going to update the `debian/copyright` file anyway, I decided to go through everything.

(regular I'm not a lawyer disclaimer)

The `README.md` states GPL-2 only, but most (or all) copyright notices in the files say:

> This program is free software: you can redistribute it and/or modify it under
> the terms of the GNU General Public License as published by the Free Software
> Foundation, either version 2 of the License, or (at your option) any later
> version.

Which implies GPL-2 or later for that particular file.

**Comment by flatcap** [comment]
> saw the changes in the copyright notice

Often, when people add code, they don't remember to credit themselves.
I finally got around to `git blame`ing the entire source tree!

> I'm packaging the new version of neomutt for Debian

Thanks!

> The README.md states GPL-2 only

Ah, thanks!  I'll tweak that to read:
"NeoMutt is released under the GPL v2+ (GNU General Public License)."

> I'm going to update the `debian/copyright` file

Ah, I had to do that for Fedora.

It was a little complicated, but the gist of it is: almost everything is **GPLv2+**
There are a few third-party contributions / inclusions:

- Source, docs and contrib: GPLv2+, except for:
- BSD: Autosetup build system, queue.h
- MIT: Acutest unit test framework, some themes
- Public Domain: pgpewrap.c, mbox.5, some themes
- License: GPLv2+ and BSD and MIT and Public Domain

Hope this helps.
